src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/client/ARNSRestClient.kt

```diff
@@ -1,18 +0,0 @@
-package uk.gov.justice.digital.hmpps.sentenceplan.client
-
-import org.springframework.cloud.openfeign.FeignClient
-import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.PathVariable
-import uk.gov.justice.digital.hmpps.sentenceplan.config.ARNSApiOauth2Config
-import uk.gov.justice.digital.hmpps.sentenceplan.data.RiskAssessment
-
-@FeignClient(
-  name = "arns-api",
-  url = "\${arns-api.base-url}",
-  configuration = [ARNSApiOauth2Config::class],
-)
-interface ARNSRestClient {
-
-  @GetMapping("/risks/crn/{crn}")
```

Comment:
do we no longer need this URL binding? or is it somewhere else and I can't see it? Ditto the GetMapping in `DeliusRestClient`.

Reply:
It's in the `ARNSApiService` now.

Reply:
oh I see yes, it's the other way around from normal `GetMapping`s. Confusing!